test/ui/text_displayer_unit.js

```diff
@@ -39,15 +41,20 @@ describe('UITextDisplayer', () => {
   }
 
 
-  beforeAll(() => {
+  beforeEach(() => {
```

Comment:
The addition of video elements to the DOM should be beforeAll instead of beforeEach.

You could move textDisplayer setup specifically to beforeEach and leave the rest in beforeAll.

Reply:
yes, you are right, thanks

was fixed